Buttons/partnershipaccept.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
         }
         if(interaction.channel.name.endsWith("- Accepted")) return interaction.reply({embeds: [embedPartnership.RequestAlreadyAccepted], ephemeral: true});
 
-        const changeEmbed = await embedPartnership.PartershipAccepted(null)
```

Comment:
The embeds should be capitalized across the files accordingly. Do not use the small first letter version.